tests/helpers/nav-auth-scenario.mjs

```
@@ -0,0 +1,248 @@
+  'minimal-sdk-absent':       { sdkScriptPresent: false, outcome: 'error', layoutDelay: 0, expected: 'skeleton', trailingWait: 3000 },
+
+  // --- #199: the SDK is late but does arrive ---
+  // Green only if a failed/hung SDK load is retried and the nav then repainted.
+  'sdk-recovers-after-hang':  { sdkScriptPresent: true,  outcome: ['hang', 'load'], trailingWait: 4000 },
+  'sdk-recovers-after-error': { sdkScriptPresent: true,  outcome: ['error', 'load'], trailingWait: 2000 },
+  // The SDK lands only AFTER initAuth stopped waiting and the page rendered.
+  // Exercises the background continuation specifically, not the inline wait.
+  'sdk-arrives-after-give-up': { sdkScriptPresent: true, outcome: ['hang', 'load'], sdkArrivalDelay: 2000, trailingWait: 5000 },
+};
+
+if (process.argv[2] === '--list') {
```

ESLint

[error] 'process' is not defined.

(no-undef)